Get "Error: Cannot find module 'nw/bin/nw'" when running Espruino Web IDE

I have installed Espruino Web IDE on Linux (Ubuntu 17.04) using the following command:

```sh
sudo npm install -g espruino-web-ide
```

However, when I run `espruino-web-ide` I get the following error:

```
module.js:327
    throw err;
    ^

Error: Cannot find module 'nw/bin/nw'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.require.resolve (internal/module.js:16:19)
    at Object.<anonymous> (/usr/lib/node_modules/espruino-web-ide/nw-loader.js:6:22)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:140:18)
    at node.js:1043:3
```

node v4.8.2
npm v2.15.11
espruino-web-ide v0.65.11

--- a/README.md
+++ b/README.md
@@ -30,6 +30,7 @@ Installing from NPM
 If you have an up to date version of [Node.js](https://nodejs.org/en/) and [NPM](https://www.npmjs.com/), you can execute the commands:
 
 * On Linux, `sudo apt-get install build-essential libudev-dev`
+* `sudo npm install nw -g`
 * `sudo npm install espruino-web-ide -g`
 * To enable BLE support: `sudo setcap cap_net_raw+eip $(eval readlink -f $(which node))`
 * `espruino-web-ide`

